proof/proof.go

```diff
@@ -7,10 +7,25 @@ package proof
 
 import (
 	"errors"
+	"hash"
 
 	"github.com/dedis/crypto"
 )
 
+// NOTE This suite is the same as the global one. Writing it is not an easy when you
+// don't know what methods you want to have (i.e.  the same set of methods that
+// edwards25519 implements etc).
```

Comment:
For the change of `crypto.v0` to `crypto.v1`, yes, this would be a bit awkward. But with the red-yellow-refactor cycle (http://blog.cleancoder.com/uncle-bob/2014/12/17/TheCyclesOfTDD.html), this is not too much of a problem. 

And changing the dependencies _does_ feel like debugging (I just got out of moving my 10'000 lines of ruby from 2.0 to 2.4... was not as bad as I thought, because I had some tests ;)